File: Program10.cpp
/*
	Write a program which accepts number from user and display digits in reverse order.
*/

#include<iostream>
using namespace std;

class Digits
{

	private:
		
		int iNum;
		
	public:
		
		Digits(int iNo)
		{
			iNum = iNo;
		}
		
		void displayReverse()
		{
			cout<<"\nDigits in reverse order are : \n";
			while(iNum != 0)
			{
				cout<<iNum%10<<"\t";
				iNum = iNum/10;
			}
			cout<<"\n";
				
		}


};

int main(void)
{

	signed int iValue = 0;
	
	cout<<"Enter number : ";
	cin>>iValue;
	
	Digits dObj(iValue);	
	
	dObj.displayReverse();
		
	return 0;
	
}

File: Program10.java
/*
	Write a program which accepts number from user and display digits in reverse order.
*/

import java.util.Scanner;

class Digits
{
	private int iNo;
	
	public Digits(int iNum)
	{
		iNo = iNum;
	}
	
	public void displayReverse()
	{
		System.out.println("\nDigits in reverse order are : \n");
		while(iNo != 0)
		{
			System.out.print(iNo%10);
			iNo = iNo/10;
		}
		System.out.println();
	}



}

class Program10
{

	public static void main(java.lang.String Arg[])
	{
		int iValue = 0;
		
		Scanner sObj = new Scanner(System.in);
		
		System.out.println("Enter Number : ");
		iValue = sObj.nextInt();
		
		Digits dObj = new Digits(iValue);
		
		dObj.displayReverse();
				
		sObj.close();
		
	}

}
